### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mtmk**

### CODE REVIEW

File: docs/documentation/intro.md [hunk 1]
```diff
@@ -1,12 +1,67 @@
...
-NATS.Net, just like NATS, is Open Source as is this documentation.
-Please [let us know](https://natsio.slack.com/channels/dotnet) if you have updates and/or suggestions for
-these docs. You can also create a Pull Request using the Edit on GitHub link on each page.
+NATS.Net, just like NATS, is open source as is this documentation.
+Please [let us know](https://natsio.slack.com/channels/dotnet) if you have updates or suggestions for
+these docs. You can also create a Pull Request in GitHub using the _Edit this page_ link on each page.
+
+## Quick Start
+
+You can [download the latest](https://nats.io/download/) `nats-server` for your platform and run it without any arguments.
```
Comment (Jarema) [inline docs/documentation/intro.md:13]: I would also mention docker image for nats-server as an alternative.

File: docs/documentation/intro.md [hunk 2]
```diff
@@ -1,12 +1,67 @@
...
+$ nats-server
+```
+
+Reference [NATS.Net](https://www.nuget.org/packages/NATS.Net) NuGet package in your project:
+
+[!code-csharp[](../../tests/NATS.Net.DocsExamples/IntroPage.cs#core-nats)]
+
+# [JetStream](#tab/jetstream)
+
+JetStream is the built-in distributed persistence system built-in to the same NATS server binary. Messages published
```
Comment (Jarema) [inline docs/documentation/intro.md:39]: repeating `build-in`

**Review by Jarema — CHANGES_REQUESTED** [review]
Some small nits only

File: docs/documentation/intro.md (hunk 1 above)
Comment (mtmk) [reply docs/documentation/intro.md:13]: added

File: docs/documentation/intro.md (hunk 2 above)
Comment (mtmk) [reply docs/documentation/intro.md]: fixed

**Review by Jarema — APPROVED** [review]
LGTM

**Pull request merged**